hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/token/TokenAccessorImpl.java

```diff
@@ -253,7 +255,9 @@ private Optional<EvmTokenInfo> getTokenInfo(final Address address) {
                 token.getSymbol(),
                 token.getName(),
                 token.getMemo(),
-                token.getTreasury().getAccountAddress(),
+                token.getTreasury().getAlias() != ByteString.EMPTY
+                        ? Address.wrap(Bytes.wrap(token.getTreasury().getAlias().toByteArray()))
```

Comment:
Avoid extra allocations. Same for other change in file.

```suggestion
                        ? Address.wrap(Bytes.wrap(DomainUtils.toBytes(token.getTreasury().getAlias()))
```